tests/src/test/resources/kg/files/linking-notfound.json

```diff
@@ -1,6 +1,6 @@
 {
   "@context" : "https://bluebrain.github.io/nexus/contexts/error.json",
-  "@type" : "ResourceNotFound",
+  "@type" : "FileNotFound",
```

Comment:
How comes that this is still a `FileNotFound` and it works. I thought I've changed all those for `ResourceNotFound`?

Reply:
Yes please. Just the JSON-LD representation has to be changed. I probably forgot about it.